Sources/MediaNavigationBar.swift

```diff
@@ -0,0 +1,128 @@
+    
+    // MARK: Instance Variables
+    weak var delegate: MediaNavigationBarDelegate?
+    
+    var hideTitleLabel: Bool {
+        didSet {
+            mediaTitleTextLabel.isHidden = hideTitleLabel
+        }
+    }
+    
+    lazy var minimizePlaybackButton: UIButton = {
+        var minButton = UIButton(type: .system)
+        minButton.addTarget(self, action: #selector(minimizePlayback), for: .touchUpInside)
+        minButton.setImage(UIImage(named: "iconChevron"), for: .normal)
+        minButton.tintColor = .white
+        minButton.translatesAutoresizingMaskIntoConstraints = false
+        return minButton
+    }()
+    
+    private var mediaTitleTextLabel: UILabel = {
+        var label = UILabel()
+        label.numberOfLines = 1
+        label.textColor = .white
+        label.font = UIFont(name: "SFProDisplay-Medium", size: 17)
+        label.text = NSLocalizedString("TITLE", comment: "Video Title")
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    lazy var chromeCastButton: UIButton = {
+        var chromeButton = UIButton(type: .system)
+        chromeButton.addTarget(self, action: #selector(toggleChromeCast), for: .touchUpInside)
+        chromeButton.setImage(UIImage(named: "renderer"), for: .normal)
+        chromeButton.tintColor = .white
+        chromeButton.translatesAutoresizingMaskIntoConstraints = false
+        return chromeButton
+    }()
+    
+    lazy var airPlayButton: UIButton = {
+        var airPlayButton = UIButton(type: .system)
+        airPlayButton.addTarget(self, action: #selector(toggleAirPlay), for: .touchUpInside)
+        airPlayButton.setImage(UIImage(named: "iconAirplay"), for: .normal)
+        airPlayButton.tintColor = .white
+        airPlayButton.translatesAutoresizingMaskIntoConstraints = false
+        return airPlayButton
+    }()
+    
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            minimizePlaybackButton.leadingAnchor.constraint(equalTo: leadingAnchor),
+            minimizePlaybackButton.topAnchor.constraint(equalTo: topAnchor),
+            minimizePlaybackButton.widthAnchor.constraint(equalToConstant: minimizePlaybackButton.frame.width),
+            minimizePlaybackButton.bottomAnchor.constraint(equalTo: bottomAnchor),
+            mediaTitleTextLabel.leadingAnchor.constraint(equalTo: minimizePlaybackButton.trailingAnchor, constant: 16),
+            mediaTitleTextLabel.topAnchor.constraint(equalTo: topAnchor),
+            mediaTitleTextLabel.trailingAnchor.constraint(equalTo: chromeCastButton.leadingAnchor, constant: 50),
+            chromeCastButton.trailingAnchor.constraint(equalTo: airPlayButton.leadingAnchor, constant: 20),
+            chromeCastButton.topAnchor.constraint(equalTo: topAnchor),
+            chromeCastButton.widthAnchor.constraint(equalToConstant: 24),
+            airPlayButton.trailingAnchor.constraint(equalTo: trailingAnchor, constant: 20),
```

Comment:
if the trailing padding at the end is 20 it should also be 20 at the beginning and since we now have 20 everywhere I would make it a let margin = 20 and replace it everywhere